### PR TITLE
[Upcoming] Release 1.1.0

### PR DESCRIPTION
**Pull request by skoudoro**
# Summary

Preparation for 1.1.0 release, targeting Monday, January 6. 

# Desired PR (need review / feedback / update)

- [x] #2021 (Submitter: @skoudoro)
- [x] #2009  (Submitter: @RafaelNH )
- [x] #2005 (Submitter: @skoudoro)

Please comment to flag any other issues or PR that should be addressed for this release

# Release checklist
- [x] add release_notes 1.1.0
- [x] Update changelog and api_changes
- [x] Update .mailmap
- [ ] Set release number in info.py and doc/conf.py
- [x] Check the copyright years in doc/conf.py and LICENSE
- [x] Check documentation
- [x] Check Tutorial
- [x] Check unit test and USE-PRE
- [x] Update index.rst
- [x] Update Highligth.rst, old_news.rst, stateofheart.rst
- [x] Update developers.rst list
- [ ] Update website
- [x] Update deprecation cycle

### CODE REVIEW

**Comment by arokem** [comment]
🎉 I'll take another look at #2009 today.

File: .mailmap [hunk 1]
```diff
@@ -99,3 +99,9 @@ Yash Sherry <yash14123@iiitd.ac.in>
 endolith <endolith@gmail.com>
 ArjitJ <32598699+ArjitJ@users.noreply.github.com>
 Kevin Sitek <sitek.kevin@gmail.com>
+Ross Lawrence <lawreros@gmail.com> Lawreros <lawreros@gmail.com>
+Ross Lawrence <lawreros@gmail.com>  Ross Lawrence <52179159+Lawreros@users.noreply.github.com>
```
Comment (Garyfallidis) [inline .mailmap]: Isn't this repeated?

File: doc/developers.rst [hunk 2]
```diff
@@ -58,9 +58,17 @@ And here is the rest of the wonderful contributors:
...
-- **Scott Trinkle**, University of Chicago
-- **Kevin R. Sitek**, MIT McGovern Institute for Brain Research
-- **Derek Pisner**, University of Texas at Austin
+- **Scott Trinkle**, University of Chicago, IL, USA
+- **Kevin R. Sitek**, MIT McGovern Institute for Brain Research, MA, USA
+- **Derek Pisner**, University of Texas at Austin, USA
+- **Ross Lawrence**, Johns Hopkins University, USA
+- **Eric Larson**, University of Washington, USA
+- **Jakob Wasserthal**, German Cancer Research Center, DE
+- **Bramsh Qamar**, Indiana University, IN, USA
```
Comment (Garyfallidis) [inline doc/developers.rst]: Chandio

File: .mailmap (hunk 1 above)
Comment (skoudoro) [reply .mailmap]: No, it is not repeated, `Lawreos` to ` Ross Lawrence` and then link the different email

**Comment by Garyfallidis** [comment]
Thank you @skoudoro! And all who contributed!

**Pull request merged**